test_constraints_min_version.txt

```diff
@@ -5,3 +5,4 @@ Pillow==4.2.1
 protobuf==3.6
 tensorflow==1.14.0
 h5py==2.9.0
+tensorboard==1.15.0
```

Comment:
Do we need this here? should be handled by setup.py.

Reply:
If we do not add it here, tensorboard version 1.14.0 is installed (because tensorflow 1.14.0 is installed here) and we do not support tensorboard 1.14.0 with torch